tests/models/test_gptq_marlin.py

```diff
@@ -63,10 +70,11 @@ def test_models(
     gptq_marlin_model = vllm_runner(model_name=model_name,
                                     revision=revision,
                                     dtype=dtype,
-                                    quantization="marlin",
+                                    quantization="gptq",
```

Comment:
@alexm-nm this test should have `marlin` for quantization

Also - is `enforce_eager=True` required?

Reply:
Oh must be a leftover from debug, good catch, will fix it in 30min

Reply:
Fixed, tests pass